calico/felix/futils.py

```diff
@@ -630,3 +631,25 @@ def find_longest_prefix(strs):
             else:
                 longest_prefix = longest_prefix[:shared_len]
     return longest_prefix
+
+def version_signal(felix_version, hostname, cluster_guid, cluster_size, cluster_type):
```

Comment:
I'd be tempted to just pass in the config rather than having lots of arguments.  We do that in a lot of places in Felix so it's more of a convention than a hard and fast rule.